vars/updatecli.groovy

```diff
@@ -4,7 +4,8 @@ def call(userConfig = [:]) {
     action: 'diff',
     config: './updatecli/updatecli.d',
     values: './updatecli/values.yaml',
-    updatecliDockerImage: 'ghcr.io/updatecli/updatecli:v0.16.0',
+    updatecliDockerImage: 'ghcr.io/updatecli/updatecli:latest',
+    containerMemory: '128Mi'
```

Comment:
should we just increase it to what you need?

Reply:
Most of the time the default 128Mi value is enought, but for [querying nginx releases for example](https://github.com/jenkins-infra/docker-confluence-data/pull/13), we have to increase it. To avoid spending too much memory everywhere, I prefer to be able to change this value only when needed.